drivers/clock_control/clock_stm32_ll_common.h

```diff
@@ -42,6 +42,33 @@
 #define z_plli2s_r(v) LL_RCC_PLLI2SR_DIV_ ## v
 #define plli2sr(v) z_plli2s_r(v)
 
+#define z_pllsai1_m(v) LL_RCC_PLLSAI1M_DIV_ ## v
+#define pllsai1m(v) z_pllsai1_m(v)
```

Comment:
For the non-L4+, `pllsai1m` should be the same as `pllm`. You can use `RCC_PLLSAI1M_DIV_1_16_SUPPORT` to discriminate